sql/views.sql

```diff
@@ -111,9 +111,10 @@ SELECT ht.schema_name AS hypertable_schema,
   cagg.user_view_name AS view_name,
   viewinfo.viewowner AS view_owner,
   cagg.materialized_only,
-  CASE WHEN mat_ht.compressed_hypertable_id IS NOT NULL 
-       THEN TRUE 
-       ELSE FALSE 
+  cagg.finalized,
```

Comment:
I fear we may be breaking some user API here. Maybe try and keep the ordering the same by putting `finalized` last? Don't know.

Reply:
It will break just for users that for some weird reason use `SELECT *...` in their applications... but for those users there are a special place reserved in the hell... heheheheheh... just kidding :-)

In fact we can't prevent users to shoot their own foot here... when we introduce the `materialized_only` column it was not added as the last column of the VIEW. For catalog tables is another history because the `attnums` we use in C code.

Reply:
Ok, you both convinced me... fixed!